### ISSUE TITLE
Import not working on osx

### ISSUE THREAD

**Issue by uday-rayala** [elarivie/pyReaderWriterLock#1]
I get an import error when importing on osx but works fine on linux.

```
>>> from readerwriterlock import rwlock
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'rwlock'
```

When I debugged further, found that there are two files in the wheel file `RWLock.py` and `rwlock.py`.  But only one of the files `RWLock.py` get unpacked in osx because file names are not case sensitive. This is not a problem on linux machines.

```
➜  Downloads unzip -l readerwriterlock-1.0.1-py3-none-any.whl
Archive:  readerwriterlock-1.0.1-py3-none-any.whl
  Length      Date    Time    Name
---------  ---------- -----   ----
     9666  07-06-2015 22:49   readerwriterlock/RWLock.py
       49  03-31-2018 01:47   readerwriterlock/__init__.py
    11569  03-30-2018 02:41   readerwriterlock/rwlock.py
     2225  04-01-2018 03:16   readerwriterlock-1.0.1.dist-info/DESCRIPTION.rst
     1066  04-01-2018 03:16   readerwriterlock-1.0.1.dist-info/metadata.json
       17  04-01-2018 03:16   readerwriterlock-1.0.1.dist-info/top_level.txt
       92  04-01-2018 03:16   readerwriterlock-1.0.1.dist-info/WHEEL
     3122  04-01-2018 03:16   readerwriterlock-1.0.1.dist-info/METADATA
      801  04-01-2018 03:16   readerwriterlock-1.0.1.dist-info/RECORD
---------                     -------
    28607                     9 files
```

Can you please help cleanup the wheel file so the library works on osx as well?

**Comment by elarivie** [comment]
@uday-rayala, Thanks for your bug report and your investigation.

This issue should now be fix with released [readerwriterlock pypi version 1.0.2](https://pypi.org/project/readerwriterlock/1.0.2/)

Could you confirm that it is fix on your side?

**Comment by uday-rayala** [comment]
@elarivie Thanks a lot for the fast reply. I can confirm that this issue no longer occurs on 1.0.2.
